samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -71,15 +71,31 @@
     // 
     "Label": "AzureSpeech",
     "Region": ""
-    // "Key":
+    // "Key": ""
+  },
+
+  "Authorization": {
+    //
+    // Authorization to use to gate access to exposed API's.
+    //  - Supported Types are "None", "ApiKey", or "AzureAd".
+    //  - Set ApiKey using dotnet's user secrets (see above) (i.e. dotnet user-secret set "ApiKey" "MY_API_KEY")
```

Comment:
The secret would need to be "Authorization:ApiKey": 
`dotnet user-secret set "Authorization:ApiKey" "MY_API_KEY"`